Mono.Cecil.Cil/MethodBody.cs

```diff
@@ -77,7 +79,7 @@ public sealed class MethodBody : IVariableDefinitionProvider {
 			get { return scope; }
 			set { scope = value; }
 		}
-
+		
```

Comment:
Looks like an unnecessary spacing change.  If we are going to upstream this PR, they may appreciate removing it.